articles/cloud-storage/cs-faq.md

```diff
@@ -139,7 +139,16 @@ Storage is billed on a per-GiB per-month basis, and is based on utilisation of t
 
 ### How is my storage bill calculated?
 
-Used storage is calculated on a daily basis. The data for a month is collected, and the average consumption calculated and rounded up to the next whole number to provide the number of GiB consumed in a month.
+Used storage is calculated on a daily basis. The data for a month is collected, and the average consumption calculated and rounded up to the next whole number to provide the number of GiB consumed in a month. The charge for the total storage is based on a tiering basis as per the Service Definition. When your storage consumption moves into the next tier, the amount of storage for that tier only is charged at the lower price.
```

Comment:
For the last sentence, is the following a bit clearer:
When your storage consumption moves into the next tier, only the amount of storage above the previous tier threshold is charged at the lower price.